.github/dependabot.yml

```diff
@@ -0,0 +1,36 @@
+version: 2
+updates:
+  # Enable version updates for Python dependencies
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    open-pull-requests-limit: 10
```

Comment:
I'm surprised that you're configuring dependabot to have a limit of open PRs. Why do you prefer it this way? And why is the number different for Pip vs GitHub Actions?